### PR TITLE
[Feature] Create openbb-cli .dockerfile and update docs

### PR DESCRIPTION
**Pull request by montezdesousa**
1. **Why**?

    - Users need to build the platform image to run it

2. **What**?

    - Uploaded image to https://github.com/orgs/OpenBB-finance/packages, update docs

3. **Impact**

    - NA

4. **Testing Done**:

    - Run the commands in the docs

### CODE REVIEW

File: build/docker/cli.dockerfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    RUN pip install /openbb/cli
+    RUN pip install openbb-devtools
+
+    # ---- Copy Files ----
+    FROM base
+
+    COPY --from=builder /usr/local /usr/local
+
+    # Launch OpenBB CLI
+    CMD ["openbb"]
```
Comment (deeleeramone) [inline build/docker/cli.dockerfile:47]: This is going to have the exact same issues that Terminal had with Docker, the window creation processes are not particularly compatible with Docker.

```
2024 Jun 05, 17:55 (🦋) /equity/price/ $ historical --symbol aapl

/usr/local/lib/python3.11/site-packages/openbb_cli/controllers/utils.py:387: FutureWarning: errors='ignore' is deprecated and will raise in a future version. Use to_numeric without passing `errors` and catch exceptions explicitly instead
  df[col] = pd.to_numeric(df[col], errors="ignore")
/usr/local/lib/python3.11/site-packages/openbb_cli/controllers/utils.py:417: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  df_outgoing = df_outgoing.applymap(str)

2024 Jun 05, 17:55 (🦋) /equity/price/ $ Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/pywry/core.py", line 349, in run_backend
    await self.runner.stdin.drain()
  File "/usr/local/lib/python3.11/asyncio/streams.py", line 392, in drain
    await self._protocol._drain_helper()
  File "/usr/local/lib/python3.11/asyncio/streams.py", line 166, in _drain_helper
    raise ConnectionResetError('Connection lost')
ConnectionResetError: Connection lost
Exception in thread Thread-1 (run):ious menu    [e] exit the program    [cmd -h] see usage and available options    Price (cmd/menu) Documentation                                               
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/pywry/core.py", line 349, in run_backend
    await self.runner.stdin.drain()
  File "/usr/local/lib/python3.11/asyncio/streams.py", line 392, in drain
    await self._protocol._drain_helper()
  File "/usr/local/lib/python3.11/asyncio/streams.py", line 166, in _drain_helper
    raise ConnectionResetError('Connection lost')
ConnectionResetError: Connection lost

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/threading.py", line 1045, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.11/threading.py", line 982, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.11/site-packages/pywry/core.py", line 384, in run
    asyncio.run(self.run_backend())
  File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pywry/core.py", line 358, in run_backend
    await self.run_backend()
  File "/usr/local/lib/python3.11/site-packages/pywry/core.py", line 339, in run_backend
    await self.runner.stdin.drain()
  File "/usr/local/lib/python3.11/asyncio/streams.py", line 392, in drain
    await self._protocol._drain_helper()
  File "/usr/local/lib/python3.11/asyncio/streams.py", line 166, in _drain_helper
    raise ConnectionResetError('Connection lost')
ConnectionResetError: Connection lost
```

**Review by deeleeramone — CHANGES_REQUESTED** [review]
CLI might not be suitable for a Docker image because it really doesn't like window creation.

**Comment by montezdesousa** [comment]
> CLI might not be suitable for a Docker image because it really doesn't like window creation.

It's possible to create windows (seen it with firefox and jupyter notebooks), sounds a good issue to open.

Meanwhile we keep the package with just non-interactive tables available or we remove it?

cc: @jmaslek

**Comment by piiq** [comment]
I vote for removing the cli docker

**Comment by deeleeramone** [comment]
> I vote for removing the cli docker

Agreed, non-interactive tables does not do anything for the charts.

**Review by deeleeramone — APPROVED** [review]

**Pull request merged**